test/src/main/java/org/apache/accumulo/test/ShellIT.java

```diff
@@ -207,9 +215,9 @@ public void addGetSplitsTest() throws IOException {
     exec("addsplits arg", false, "java.lang.IllegalStateException: Not in a table context");
     exec("createtable test", true);
     exec("addsplits 1 \\x80", true);
-    exec("getsplits", true, "1\n\\x80");
+    exec("getsplits", true, "1\\x80");
     exec("getsplits -m 1", true, "1");
-    exec("getsplits -b64", true, "MQ==\ngA==");
+    exec("getsplits -b64", true, "MQ==gA==");
```

Comment:
If these tests pass without the newline, then that seems like it would indicate a breaking change to getsplits.

Reply:
Same issue as above. Printed instead of PrintLn